src/components/control_form/index.js

```diff
@@ -4,27 +4,48 @@ import { connect } from 'react-redux';
 import { selectCurrentPalette } from '../../actions';
 
 export class ControlForm extends Component {
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
     this.state = {
-      project_name: '',
+      project_name: this.setName(),
       project_palettes: []
     }
   }
 
   componentDidMount() {
-    let { palettes, currentProject } = this.props
+    // let { palettes, currentProject } = this.props;
```

Comment:
Do we have a plan for this componentDidMount? do we need it anymore?

Reply:
I fixed that on the [Rebase PR](https://github.com/ehk9000/Palette-Picker-UI/pull/50)